src/main/java/com/vinylteam/vinyl/security/SecurityConfiguration.java

```diff
@@ -50,14 +56,26 @@ public void configure(WebSecurity web) throws Exception {
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
+                .cors(withDefaults())
                 //.sessionManagement()
                 //.sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .authorizeRequests()
                 .anyRequest().permitAll()
                 .and()
-                .addFilterAfter(userAttributeFilter, UsernamePasswordAuthenticationFilter.class)
                 .addFilterBefore(jwtValidatorField, UsernamePasswordAuthenticationFilter.class)
-                .formLogin().loginPage("/signIn").permitAll().usernameParameter("email").defaultSuccessUrl("/profile");/*.successHandler()*///;
+                .formLogin().loginPage("/signIn").permitAll().usernameParameter("email").defaultSuccessUrl("/profile")
+                .and()
+                .logout().addLogoutHandler(logoutHandlerService).logoutSuccessUrl("/successlogout");
```

Comment:
we have /logout set by Spring Security

Reply:
do we actually need a success URL for that functionality? We are REST service

Reply:
yes, we do...this controller method returns the appropriate answer after Spring Security did its logout magic.
I see shortcoming it this decision, since the endpoint is exposed and hence can be requested from outside but this call will not make any sense.
Could you please advise whether there is a better way to do the same thing?
 TIA

Reply:
issue was created

https://app.clickup.com/t/u3xqgm